.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+name: Docker Plugin Publish
+on:
+  workflow_dispatch: null
+  release:
+    types: [ published ]
+jobs:
+  oci_publish:
+    name: Build and publish the docker plugin
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v3
+
+      - name: setup python 3
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push to DockerHub
+        run: make deploy
```

Comment:
Just as a redundancy to ensure the released image tag matches the corresponding GitHub release, could we override the `PLUGIN_VERSION` with the `github.event.release.tag_name` variable? 
Realistically I don't think this would change anything since the deployed version is already derived from the latest local git tag, but it might be nice to have some extra confidence 👍